.github/workflows/build-and-test.yml

```diff
@@ -141,6 +141,8 @@ jobs:
         flags: funTest-non-docker
   funTest-docker:
     runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/oss-review-toolkit/ort
```

Comment:
Show we specify the last nightly image or the latest released ? Somehow tests that depends on new features on base components that are not available on the curren published ones.

Reply:
This should be as recent as possible, i.e. a daily build tag. Anyway, for testing I was trying to use `latest` (both implicitly by not specifying a tag, and by explicitly specifying the tag name), but for some reason it [does not work](https://github.com/oss-review-toolkit/ort/actions/runs/6855036442/job/18639246562?pr=7853#step:2:21).

Reply:
Seems to be related to https://github.com/docker/build-push-action/issues/820.

Reply:
> This should be as recent as possible, i.e. a daily build tag

Using the `snapshot` Docker tag now for that matter.